components/TripDetails/index.js

```diff
@@ -0,0 +1,28 @@
+import Image from "next/image";
+import Link from "next/link";
+
+export default function TripDetails({
+  image,
+  title,
+  city,
+  country,
+  startDate,
+  endDate,
+  description,
+}) {
+  return (
+    <>
```

Comment:
I suggest to make this a `main` element as your details page does not have one yet. The `main` element contains the main content of your page.
```suggestion
    <main>
```